babel.config.blocks.js

```diff
@@ -3,7 +3,6 @@ const config = {
 		[
 			'@babel/env',
 			{
-				targets: { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] },
```

Comment:
why we don't need this?

Reply:
It uses the browserlist defined elsewhere:

https://github.com/Automattic/jetpack/blob/808170fbe623b720fea37d04a377137b6b448c55/package.json#L13-L15

Reply:
It's basically duplicate of the browserslist config here:

https://github.com/Automattic/jetpack/blob/48bd35a11c690c8bd8abf398ac5e324ad1d708c8/package.json#L13-L15

The same config got removed from Calypso Babel just recently as well, for the same reason: https://github.com/Automattic/wp-calypso/pull/31561

As a result browser compatibility requirement are defined in one place only. `:success:`